src/options/options.html

```diff
@@ -207,6 +207,7 @@ <h1 data-i18n="__MSG_titleAddonBehavior__">Addon behavior</h1>
 			</section>
 
 			<section>
+				<hr />
```

Comment:
Okay, for some reason this does seem to shrink the size of the button, does not it?

It is so "flat" now... (in height)
![grafik](https://user-images.githubusercontent.com/11966684/55228824-672b5b00-521b-11e9-840c-cba44ae4256f.png)

Maybe this can be changed?

Also, generally, do you like the seperation line there, or do you think it is not really needed and just confuses users?

Reply:
do you think its better to use a br instead? just put more padding to show that its different?

Reply:
I honestly am not sure. Maybe try and see whether it looks better…

Note that in any case you can keep the CSS for the `hr`… Maybe it can be used later anyway… :wink: 

Note I am also thinking about https://github.com/rugk/offline-qr-code/issues/173 and there we could have some more content at the bottom, which may look totally different then…

So I am open to suggestions/ideas here… :smiley:

Reply:
i changed it to 2 `br` looks not too bad